chart/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
-appVersion: "1.8"
-version: "1.1.3"
+appVersion: "1.9"
+version: "1.2.0"
```

Comment:
This was presumptuous on my part based on repo activity and no in-flight PRs.

Reply:
All good